SuperRonanCraft/bootcamp-Tomodachi
# Render pages

## Purpose

Please provide a detailed explanation of the PR. What existing problem does it solve?

Fixes #25 

## Changes


- path/to/file
  - client/src/App.jsx
  - client/src/components/PageHandler.jsx
  - client/src/main.jsx

## Steps to Reproduce or Test


- clone the branch
- npm i
- npm run dev
- should see a placeholder message for each page change the default state to go to each page.
